docassemble/AssemblyLine/data/questions/al_saved_sessions.yml

```diff
@@ -1,267 +1,151 @@
 ---
-variable name: al_sessions_url_ask_snapshot
-data:
-  - undefine: 
-    - al_sessions_save_session_snapshot
-    - al_sessions_new_session_id
-  - al_sessions_snapshot_label
-  - al_sessions_save_session_snapshot
-  - al_sessions_snapshot_results
----
-variable name: al_sessions_url_ask_fast_forward
-data:
-  - undefine:
-    - al_sessions_new_session_id
-  - al_sessions_source_session
-  - al_sessions_fast_forward_session
-  - al_sessions_launch_new_session
----
-template: al_sessions_preview_variables
-subject: |
-  Preview
-content: |
-  <pre>
-  ${ verbatim(json.dumps(al_simple_filtered_vars, sort_keys=True, indent=4)) }
-  </pre>  
+modules:
+  - .sessions
 ---
 id: save answer snapshot
 # Save a copy
-comment: |
-  Copy the answers from the "current session" into the 
-  "al_saved_session_store.yml" interview.
+decoration: save
 question: |
-  Save answer snapshot
+  Save an answer set
 subquestion: |
-  You have visited ${ all_variables(include_internal=True).get('_internal',{}).get('steps',1) }
-  screens so far and defined ${ len(al_sessions_filtered_vars) } fields.
+  % if not user_logged_in():
+  <div class="alert alert-danger fade show d-flex" role="alert">
+  :exclamation-triangle: &nbsp;
+  <div>
+  You need to be signed in to save an answer set.
+  </div>
+  </div>
   
-  ${ collapse_template(al_sessions_preview_variables) }
+  The "answer set" feature lets you make a copy of your answers to
+  use again and again. You can load this answer set in a new interview later.
+  
+  ${ action_button_html(url_of('login'), label="Sign in", icon="sign-in-alt", size="md", color="primary") }
+  % else:
 
-  Save a copy of your current answer progress, including any files, to load into
-  this interview or a different interview later.
+  <div class="alert alert-info alert-dismissible fade show d-flex" role="alert">
+  :info-circle: &nbsp;
+  <div>
+  Clicking the "next" button automatically saves your progress. You can
+  <a href="${ url_of('interviews') }" class="alert-link">visit in-progress forms</a>
+  to keep working on a form that you started earlier.
+  </div>
+  <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+  </div>
```

Comment:
I like the popup too!

How about:
  "You do not need to use this feature to save your progress, docassemble automatically saves that."

This avoids the confusion about the "next button", also because I found the interviews screen link in the msg kind of distracting.